jupyter-extension/jupyterlab_pachyderm/pfs_manager.py

```diff
@@ -699,7 +640,7 @@ def _get_file_from_path(self, path: str) -> pfs.File:
         else:
             pach_path = str(Path(*parts[1:]))
 
-        file_uri = f"{commit_uri}:{pach_path}"
+        file_uri = f"{branch.as_uri()}:{pach_path}"
```

Comment:
We can remove `commit_uri` a few lines above since it's no longer used

Reply:
Ah actually I should not have changed this method to add branch. DatumManager should remain the same. I will just revert adding the param for this method.